File: CursoJava/Repetição-Sequencial-Condicional/src/Exer/EstruturaSequencial/Exer04.java
package Exer.EstruturaSequencial;

import java.util.Scanner;

public class Exer04 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		System.out.print("Digite o ID do funcionário: ");
		int id = sc.nextInt();
		System.out.print("Digite o número de horas trabalhadas: ");
		double hours = sc.nextDouble();
		System.out.print("Digite o valor da hora trabalhada: ");
		double cash = sc.nextDouble();
		
		double salary = hours * cash;
		
		System.out.println("------------");
		System.out.println("ID: " + id);
		System.out.println("Salário: U$ " + salary);
		
		sc.close();
	}

}

File: CursoCSharp/Exer/EstruturaSequencial/Exer04.cs
using System;

namespace exer.udemy.EstruturaSequencial
{
    public class Exer04
    {
        static void Main(string[] args)  
        {
            Console.WriteLine("Informe o número do funcionário: ");
            int id = int.Parse(Console.ReadLine());

            Console.WriteLine("Informe o número de horas extras: ");
            double hours = double.Parse(Console.ReadLine());

            Console.WriteLine("Informe o valor por hora trabalhada: ");
            double value = double.Parse(Console.ReadLine());

            double salary = hours * value;

            Console.WriteLine($"ID: {id}");
            Console.WriteLine("Salário = R$" + salary.ToString("F2", CultureInfo.InvariantCulture));
        }      
    }
}